docs/reference/constants.html

```diff
@@ -48,6 +48,11 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 </script>
 
+It must be noted that without parameter these constant are proper function objects and can be used in
```

Comment:
+ the term is 'Callable Object'
 + it is displayed before so this is redundant info.
 + the fact they can be pass for optimization will be betttr put on function pages
 
 So I think this need to be rewritten or removed